plugins/carbon_black_response/plugin.spec.yaml

```
@@ -971,10 +1007,11 @@ actions:
     description: Updates or Resolves an Alert in Carbon Black
     input:
       id:
-        description: 'Unique ID of the alert. Example: 1cb11d0d-f86b-415d-aeb3-05f085973fbb'
+        description: 'Unique ID of the alert. '
```

Just noticed a small trailing space, but happy to approve.

```suggestion
        description: 'Unique ID of the alert.'
```